python_frank_energie/exceptions.py

```
@@ -1,17 +1,34 @@
+    """Request failed."""
+
+class FrankEnergieError(Exception):
+    """Base class for all FrankEnergie-related errors."""
+
+class LoginError(FrankEnergieError):
+    """Raised when login to FrankEnergie fails."""
+
+class NetworkError(FrankEnergieError):
+    """Raised for network-related errors in FrankEnergie."""
+
+class RequestException(Exception):
```

Ruff

Redefinition of unused `RequestException` from line 12

(F811)